### ISSUE TITLE
Use TouchableHighlight instead of TouchableOpacity

### ISSUE THREAD

**Issue by Liampronan** [tongyy/react-native-draggable#18]
so that image doesn't blink when clicked. I can create a PR for this if it sounds good. could also make it an option

**Comment by tongyy** [comment]
sure, it sounds good! the default "underlayColor" is black. if you can change default is "none" and add prop of underlayColor for react-native-draggable, I will accept it and publish to the new version of NPM.

**Comment by MartinSchere** [comment]
This wasn't implemented yet, right?

**Comment by baconcheese113** [comment]
Nope, instead I allowed props to be passed through to it. So you can set `activeOpacity`
https://reactnative.dev/docs/0.8/touchableopacity#activeopacity

See touchableOpacityProps here https://github.com/tongyy/react-native-draggable#properties
